food-chain/food_chain_test.rb

```diff
@@ -15,14 +24,38 @@ class FoodChainTest < Minitest::Test
   # specifically want feedback on them.
   def test_the_whole_song
     song_file = File.expand_path('../song.txt', __FILE__)
-    expected = IO.read(song_file)
+    expected  = IO.read(song_file)
```

Comment:
Probably only there for visual alignment.  Not as apparent if there were 3 or more.